.editorconfig

```diff
@@ -26,5 +26,11 @@ ij_java_imports_layout = $*,|,*
 ij_java_names_count_to_use_import_on_demand = 1000
 ij_java_wrap_comments = true
 
+[*.py]
```

Comment:
We didn't have this section before so intellij was using 2 space indent instead of PEP8's 4.